packages/hint-sri/src/types.ts

```diff
@@ -4,3 +4,8 @@ export enum algorithms {
     sha384 = 2,
     sha512 = 3
 }
+
+export enum originCriteria {
```

Comment:
For consistency, the name of the `enum` should be Pascal Case.

Reply:
The one above is `algorithms` doesn't follow it neither. I'll update it as well.

Reply:
Done

Reply:
Thanks!